src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -34,10 +34,11 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public abstract class CollectionItemsEqualConstraint : CollectionConstraint
     {
+
```

Comment:
This is one of those things we don't like merging because of the git blame for that area.

Reply:
Fair enough, should have been more careful in reviewing.